### ISSUE TITLE
wrong interpulation of key with ":)"

### ISSUE THREAD

**Issue by felixmosh** [i18next/i18next#1456]
## 🐛 Bug Report

```javascript
const value = t("Welcome to React :)");
console.log(value) // will print ")"
```

## To Reproduce

https://codesandbox.io/s/react-i18next-cmgdb?file=/src/app.js:213-237

## Expected behavior

It should return the **original key** if translation not found.

## Your Environment

- *runtime version*: i.e. node v12, deno, browser xy
- *i18next version*: >=19.0.0
- *os*: Mac, Windows, Linux
- *any other relevant information*

**Comment by felixmosh** [comment]
From debugging the code it seems that it is related to https://github.com/i18next/i18next/blob/605cebb0a2095d029e1261f8d12b7997716d5187/src/Translator.js#L46

**Comment by adrai** [comment]
set nsSeparator to false

**Comment by felixmosh** [comment]
Ha, cool, I can pass locally `const value = t("Welcome to React :)", {nsSeparator: false});`

Thank you 🙏🏼

**Comment by adrai** [comment]
yes, works locally or globally, on i18next.init()

**Comment by felixmosh** [comment]
Hi, passing `nsSeparator=false` won't work because there is a [logical or](https://github.com/i18next/i18next/blob/605cebb0a2095d029e1261f8d12b7997716d5187/src/Translator.js#L46), which won't accept `false`.

Is this a bug?

**Comment by adrai** [comment]
You mean when used directly in the t function? Yes, this is a bug.

**Comment by adrai** [comment]
will fix it asap

**Comment by felixmosh** [comment]
Will you accept a PR? :]

**Comment by adrai** [comment]
Yes, but I already released a new version with the fix ;-) => 19.5.6

**Comment by adrai** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project.

There are many ways to help this project 🙏
